lib/omniauth/strategies/telegram.rb

```diff
@@ -13,10 +13,9 @@ class Telegram
       option :bot_name, nil
       option :bot_secret, nil
       option :button_config, {}
-      
-      FIELDS      = %w[id first_name last_name username photo_url auth_date hash]
-      HASH_FIELDS = %w[auth_date first_name id last_name photo_url username]
-      
+
+      REQUIRED_FIELDS = %w[id username auth_date hash]
```

Comment:
Currently I don't get the `photo_url` param for my telegram account. So I simplified checking params.